docs/docs/contributions/releases/release-process.mdx

```diff
@@ -97,33 +97,7 @@ $ git checkout -b 2.9.x
 $ git push upstream 2.9.x
 ```
 
-## Step 2: Update this docs site
```

Comment:
In a separate PR I'll edit the `release` workflow to trigger a docs sync at the end of a release.